### PR TITLE
Add palindrome

### PR DESCRIPTION
**Pull request by muthuannamalai12**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?
Yes
(Write your answer here.)

### Description
Added a Python Program to check whether a number is palindrome or not
(Write your answer here.)

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests
#2044
(Write your answer here.)

### CODE REVIEW

File: Python/other/Palindrome.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
+/*
+Python Program to check whether a number is Palindrome or not
+An integer is a palindrome if the reverse of that number is equal to the original number.
+Example of Palindromes are 11,44,101,121.
+*/
```
Comment (HarshCasper) [inline Python/other/Palindrome.py]: Python does not use this. You can't be more serious using them in Python Code

File: Python/other/Palindrome.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+Enter a number: 121
+121 is a palindrome
+
+Example 2:
+Enter a number: 123
+123 is not a palindrome
+
+Time Complexity: O(n)
+Space Complexity: O(1)
+*/
```
Comment (HarshCasper) [inline Python/other/Palindrome.py]: Using `/*.....*/` in a Python Code?

File: Python/other/Palindrome.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
+/*
+Python Program to check whether a number is Palindrome or not
+An integer is a palindrome if the reverse of that number is equal to the original number.
+Example of Palindromes are 11,44,101,121.
+*/
+
+def palindrome(n):
+    a = n
+    sum = 0
+    r = 0
```
Comment (HarshCasper) [inline Python/other/Palindrome.py]: Use meaningful variable names

File: Python/other/Palindrome.py [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    while n > 0:
+        r = n % 10
+        sum = sum * 10 + r
+        n = n // 10
+    if sum == a:
+        print (a, 'is a palindrome')
+    else:
+        print (a, 'is not a palindrome')
+n = eval(input('Enter No '))
+palindrome(n)
```
Comment (HarshCasper) [inline Python/other/Palindrome.py]: Reference this in `if __name__ == '__main__'` block

**Review by HarshCasper — CHANGES_REQUESTED** [review]

File: Python/other/Palindrome.py (hunk 1 above)
Comment (muthuannamalai12) [reply Python/other/Palindrome.py]: Sorry for the inconvenience. I will correct it

File: Python/other/Palindrome.py (hunk 2 above)
Comment (muthuannamalai12) [reply Python/other/Palindrome.py]: Sorry for Inconvenience

**Comment by muthuannamalai12** [comment]
I have updated kindly review again @HarshCasper @atarax665

**Comment by muthuannamalai12** [comment]
I have updated kindly review again @HarshCasper @atarax665

File: Python/README.md [hunk 5]
```diff
@@ -258,5 +258,6 @@ _add list here_
 - [Longest substring without repeating character](other/longest_Substring.py)
 - [Dutch National Flag Algorithm Implementation in Python](other/SortColors-DutchNationalFlag.py)
 - [Ploting Diffrent Graphs in Python](other/Plot_Graph.py)
+- [Palindrome](other/Palindrome.py)
```
Comment (atarax665) [inline Python/README.md]: This program should be added in Math folder. Update the readme accordingly.

**Review by atarax665 — CHANGES_REQUESTED** [review]

**Comment by muthuannamalai12** [comment]
I have updated it. Kindly Review again @atarax665

**Comment by muthuannamalai12** [comment]
I have updated it. Kindly Review again @atarax665

**Comment by muthuannamalai12** [comment]
I have updated it. Kindly Review again @atarax665

**Review by HarshCasper — APPROVED** [review]

**Comment by muthuannamalai12** [comment]
@HarshCasper @atarax665. Can you please add DWOC and level label for this issue here on my PR.

**Comment by muthuannamalai12** [comment]
@HarshCasper @atarax665 @anushkrishnav . Can you please add DWOC and level label for this issue here on my PR.

**Comment by muthuannamalai12** [comment]
@HarshCasper @atarax665 @anushkrishnav . Can you please add DWOC and level label for this issue here on my PR.

**Pull request merged**